### DISCUSSION TITLE
Custom trained coral models usage

### DISCUSSION THREAD

**Discussion by liri** [blakeblackshear/frigate#2017] [Ask A Question]
Hi and thanks for helping !
I wonder if there is any custom model which is better to use with frigate from the list below:
https://coral.ai/models/object-detection/
(assuming i have a google coral stick)

If so - how do i map a volume to use one of these?

Thanks again

**Comment by trythatagain** [comment]
Most of those model seem the same but with certain objects removed. Not sure what difference that makes though.
It would be nice to find a list of alternative models with different objects, e.g people, cars and more animals.
I had a look in the container and they are in the root '/' folder as stated in the docs [here](https://docs.frigate.video/configuration/objects/)
so I assume you would use something like this under 'volumes:',
'- /path/to/your/storage/edgetpu_new_model.tflite':/edgetpu_model.tflite', and 
'- /path/to/your/storage/new_labelmap.txt:/labelmap.txt'  if using a edgetpu.

**Reply by liri** [reply]
thanks, will try using one of these models. I see that it supports 90 objects which is a larger number than the list frigate provides in the config i think.

**Reply by hendrikwegner** [reply]
Frigate uses a model trained on the COCO Dataset which also supports 90 objects. In the labelfile are 10 positions with 'n/a' and the rest is identical with the list from the docs.

**Reply by liri** [reply]
ok we're making a progress :)
from all the COCO models listed under coral website, isn't there a better one to be used by frigate under NUC?

**Reply by hendrikwegner** [reply]
I would assume that the model is not so much the issue but rather the data it was trained on and the environment you are trying to use it in. For example the COCO Dataset was not intended to be used for CCTV Object detection since the camera angle and positioning differ much from the training data (explore here: https://cocodataset.org/#explore). In my testcase (CCTV) outside of Frigate I couldn't observe a significant detection-increase with EfficientDet-Lite3 (the largest model I could load with my coral usb accel.) compared to the SSD MobilNet V2 (inference speeds of 220ms compared to 12ms).

The other day I saw a thread here on github from the Frigate-Maintainer for the interest of a specially trained model for CCTV use but I can't find it right now.. All things being said coming from a total rookie

**Reply by ozett** [reply]
https://github.com/blakeblackshear/frigate/discussions/1043#discussioncomment-1522414
